client/state/comments/reducer.js

```diff
@@ -50,15 +50,35 @@ export const deconstructStateKey = key => {
 	return { siteId: +siteId, postId: +postId };
 };
 
+const isCommentManagementEdit = newProperties =>
+	has( newProperties, 'commentContent' ) &&
+	has( newProperties, 'authorDisplayName' ) &&
+	has( newProperties, 'authorUrl' );
+
 const updateComment = ( commentId, newProperties ) => comment => {
```

Comment:
I think this function has grown to the point that it justifies having a test suite, specially with the split behaviour of edit vs like/status change.

Reply:
Good point!
Added a new test case targeting the three Comment Management fields (`commentContent`, `authorDisplayName`, `authorUrl`) in f175eef08473d3a49ee197441365e0c60d9fc53d